README.md

```diff
@@ -203,6 +203,7 @@ All the data (past and coming) is available publicly in JSON:
 * 30: [PrestaShop Developer Conference](https://events.prestashop.com/prestashop-developer-conference) - Paris (France) <a href="https://hopscotch.key4events.com/abstract.aspx?e=321&c=2683"><img alt="CFP PrestaShop Developer Conference" src="https://img.shields.io/static/v1?label=CFP&message=until%2029-September-2023&color=green"></a>
 * 30: [WHO run the Tech](https://whorunthetech.com/) - Rennes (France) <a href="https://conference-hall.io/public/event/Pe62MIv1mSH9KF7nDGRA"><img alt="CFP Who Run The Tech #8" src="https://img.shields.io/static/v1?label=CFP&message=until%207-September-2023&color=green"></a>
 * 30: [DevFest Stockholm](https://www.devfest-stockholm.com/) - Stockholm (Sweden) <a href="https://sessionize.com/devfest-stockholm-2023/"><img alt="CFP DevFest Stockholm 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2030-September-2023&color=green"></a>
+* 30-31: [Asynconf Event](https://www.asynconf.fr/) - Paris + Remote <a href="https://asynconf.fr/"><img alt="Asynconf Event 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2030-October-2023&color=green"></a>
```

Comment:
Quel est le lien pour soumettre un talk au CFP (call for paper) et quelle est la date de fin pour soumettre un talk ?
S'il n'y a pas de CFP ouvert et accessible, alors pas besoin de mettre le lien et la balise "a" :-)

Reply:
CFP enlevé